requirements-extras.txt

```
@@ -1,5 +1,5 @@
 qiskit-ibm-provider>=0.6.1 # for submitting experiments to backends through the IBM provider
 cvxpy>=1.3.2 # for tomography
 scikit-learn # for discriminators
-qiskit-aer>=0.11.0 # for QV simulations
+qiskit-aer<=0.12.2 # temp fix.
```

```suggestion
qiskit-aer>=0.11.0,<=0.12.2 # temp fix.
```
Any reason to remove the lower bound?

---

I made an issue so now the comment could be more specific:

```suggestion
qiskit-aer>=0.11.0,<=0.12.2 # Temporary version pin because of https://github.com/Qiskit-Extensions/qiskit-experiments/issues/1292
```

---

But hopefully we can remove the pin soon :slightly_smiling_face: